Cargo.toml

```diff
@@ -38,7 +38,7 @@ sha2 = { version = "0.10", optional = true }
 blake3 = { version = "1.4.0", optional = true, features = ["traits-preview"] } 
 hex = "0.4"
 tracing = "0.1"
-ics23 = { version = "0.11.0", optional = true}
+ics23 = { version = "0.12.0", optional = true}
```

Comment:
`tonic` bump LGTM: https://github.com/cosmos/ics23/releases/tag/rust%2Fv0.12.0

Reply:
I am not sure what that means. Did you mean that upstream also bumped?